Rubberduck.Core/Common/WinAPI/RegistryAccess.cs

```diff
@@ -18,9 +18,11 @@ internal static RegistryKey GetDeviceKey(string device)
 
         internal static string GetClassType(string classGuid)
         {
-            var classGuidKey = Registry.LocalMachine.OpenSubKey(@"SYSTEM\CurrentControlSet\Control\Class\" + classGuid);
-
-            return classGuidKey != null ? (string)classGuidKey.GetValue("Class") : string.Empty;
+            using (var classGuidKey =
+                Registry.LocalMachine.OpenSubKey(@"SYSTEM\CurrentControlSet\Control\Class\" + classGuid))
```

Comment:
This isn't safe. `OpenSubKey` can return `null`.

Reply:
This is another case of IDisposable analyers at work. I was surprised to see those not being disposed -- in fact, I was surprised that `Registry` stuff were disposable. As such, I wasn't really thinking too much about the possibility of returning `null`. But that is obviously problematic. I will revise those.

Reply:
Ref https://github.com/DotNetAnalyzers/IDisposableAnalyzers/issues/128